### PR TITLE
Docs target

### PR DESCRIPTION
**Pull request by jamesaimonetti**
Checks the mkdocs.yml for errors and fails the build if found

### CODE REVIEW

File: scripts/validate_mkdocs.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        return parse_page_string(errors_detected, page)
+    else:
+        return errors_detected
+
+stream = open("doc/mkdocs/mkdocs.yml", 'r')
+mkdocs = yaml.load_all(stream)
+errors_detected = False
+
+for doc in mkdocs:
+    for k,v in doc.items():
```
Comment (fenollp) [inline scripts/validate_mkdocs.py]: This looks like python3. Kazoo asks for python2 in the docs: should fix one of the two

Comment (jamesaimonetti) [inline scripts/validate_mkdocs.py]: weird, I am using python 2.7.12+

**Comment by jamesaimonetti** [comment]
@fenollp I think the circle.yml was misconfigured. It seems to have run properly now.

**Comment by fenollp** [comment]
```
make docs
./scripts/validate_mkdocs.py
Traceback (most recent call last):
  File "./scripts/validate_mkdocs.py", line 4, in <module>
    import yaml, os.path, sys
ImportError: No module named yaml
make: *** [docs] Error 1
```

**Pull request merged**